german/the-black-hack.md

```diff
@@ -46,9 +46,9 @@ Rüstung bietet Schutz, indem sie allen erlittenen Schaden reduziert. Jedes Stü
 
 |     Typ        | Punkte |
 |:--------------:|:------:|
-|  Festes Wams   |   2    |
-|  Lederrüstung  |   4    |
-|   Kettenhemd   |   6    |
+| Gambeson       |   2    |
```

Comment:
Ja so wird ein Schuh draus. Also mit Wams kann ich leben. Es ging mir nur darum dieses In-Group Begrifflichkeiten Argument zu verwerfen.
Klar gibt es keinen wirklichen Unterschied zwischen Wams und Gambeson. Das Sarrock noch mehr rausfällt sind wir uns auch einig.
Das der Duden Wams kennt und es auch seit jeher zum üblichen Sprachgebrauch gehört spricht natürlich absolut für das Wams.  Mir ging es ja einfach nur um darum, dass Suchende direkt eine hilfreiche Erklärung finden.
Wenn sie dies über Wams machen kommt ja zum Glück als erstes die richtige Definition und ein paar Einträge drunter erst die Welt am Sonntag. xD
Sucht man nach Festes Wams kommt nur Murks bei rum. Einfach nur Wams läuft auch mit mir konform.